Fix missing definition of uint64_t while compiling
under uclibc or glibc

https://bugs.launchpad.net/hplip/+bug/1826965

--- a/scan/sane/OrbliteScan/LinuxCommon.h
+++ b/scan/sane/OrbliteScan/LinuxCommon.h
@@ -2,6 +2,7 @@
 #define H_LinuxCommon
 
 #include <sys/types.h>
+#include <stdint.h>
 
 /* Common typedefs for Linux */
 
